remoulade/broker.py

```diff
@@ -305,7 +308,17 @@ def enqueue(self, message: "Message", *, delay: Optional[int] = None) -> "Messag
         Returns:
           Message: Either the original message or a copy of it.
         """
-        raise NotImplementedError
+
```

Comment:
maybe add a function to manage delay before this to be consistent with what we had before. Is it important @antoinerabany?